packages/loaders/src/Spinner.example.md

```diff
@@ -0,0 +1,137 @@
+The `<Spinner />` component includes several accessibility
+and usability features:
+
+- Inherits the `font-size` and `color` properties
+  from it's parent element and additionally allows local overrides
+- Delays the initial render of the component to reduce UI shifting
+  in normal loading conditions
+- Includes accessibility attributes to identify as an `indeterminate progress bar`
+
+### Basic Usage
+
+```jsx
+const { zdColorBlue500 } = require('@zendeskgarden/css-variables');
+
+<Spinner color={zdColorBlue500} size="50px" />;
```

Comment:
[nit] please update all examples from current `50px` to `48px` for base-4 consistency :wink: